pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -142,7 +146,7 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *v1.Pod, attemp
 	}
 	podSandboxConfig.Linux = lc
 
-	if runtime.GOOS == "windows" {
+	if runtime.GOOS == os_windows {
```

Comment:
We can use this one:
https://github.com/kubernetes/kubernetes/blob/487c44323922fa9a0d217ac3a70eb5943a121fbb/pkg/apis/core/types.go#L3115

Reply:
Thanks, updated.